packages/discord.js/src/managers/MessageManager.js

```diff
@@ -266,6 +266,38 @@ class MessageManager extends CachedManager {
 
     await this.client.rest.delete(Routes.channelMessage(this.channel.id, message));
   }
+
+  /**
+   * Ends a poll.
+   * @param {Snowflake} channelId The id of the channel
+   * @param {Snowflake} messageId The id of the message
+   * @returns {Promise<Message>}
+   */
+  async endPoll(channelId, messageId) {
+    const message = await this.client.rest.post(Routes.expirePoll(channelId, messageId));
+    return this._add(message, false);
+  }
+
+  /**
+   * Options used for fetching voters of an answer in an uncached poll.
+   * @typedef {FetchPollAnswerVotersOptions} FetchUncachedPollAnswerVotersOptions
+   * @param {Snowflake} channelId The id of the channel
+   * @param {Snowflake} messageId The id of the message
+   * @param {number} answerId The id of the answer
+   */
```

Comment:
Is it possible to rename this to something akin to `BaseFetchOptions` -> `FetchMessageOptions`? Maybe `BaseFetchPollAnswerVotersOptions` -> `FetchPollAnswerVotersOptions`? If not, then that's alright too.